packages/backend/src/graphql/invoice/invoice.ts

```diff
@@ -10,13 +11,25 @@ export const Invoice = builder.prismaObject('Invoice', {
     invoiceDate: t.expose('invoiceDate', { type: 'Date' }),
     customerAddress: t.exposeString('customerAddress', { nullable: true }),
     customerName: t.exposeString('customerName'),
-    organization: t.relation('organization'),
-    invoiceItems: t.relation('InvoiceItems', {
-      description: 'Items associated with the invoice',
-      resolve: (query, invoice) => {
-        return prisma.invoiceItem.findMany({ ...query, where: { invoiceId: invoice.id }, orderBy: { start: 'asc' } })
+    payDate: t.expose('payDate', { type: 'Date', nullable: true }),
+    sendDate: t.expose('sendDate', { type: 'Date', nullable: true }),
+    invoiceStatus: t.field({
+      type: InvoiceStatusEnum,
+      description: 'Status of the invoice',
+      select: { invoiceStatus: true, payDate: true, sendDate: true },
+      resolve: (invoice) => {
+        const now = new Date()
+        if (invoice.payDate && invoice.payDate < now) {
+          return InvoiceStatus.PAID
+        }
+        if (invoice.sendDate && invoice.sendDate < now) {
+          return InvoiceStatus.SENT
+        }
+
+        return InvoiceStatus.DRAFT
```

Comment:
Should we also check if `payDate` is earlier than `sendDate` and throw an error?

Reply:
Implemented an error message if payDate is earlier than sendDate.